.github/workflows/release-kotlin.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           rustup toolchain install nightly
 
           # Update both toolchains
-          rustup update stable
+          rustup update nightly-2023-11-24
```

Comment:
Sadly needed to do this due to https://github.com/rust-lang/rust/issues/65991

Reply:
If we switching back to nightly, then I think it would be good to revert `rust-toolchain.toml` file.

Reply:
Sounds good, will also add a comment describing why we needed this for this particular PR and when what features need to be stable before we can switch to stable.